src/test/kotlin/com/kafkakotlin/demo/users/UserServiceTest.kt

```diff
@@ -0,0 +1,33 @@
+// package com.kafkakotlin.demo.users
```

Comment:
Any reason this is commented out? seems a valid test to have

Reply:
Another *shame* moment. Added it back in and updated in line with UserControllerTest.kt